### PR TITLE
add nx comparison

### LINKED ISSUE #20: [FEA] Upgrade Pagerank tests to compare against Nx reference.
We need a baseline to validate the results

### PR DESCRIPTION
**Pull request by Iroy30**
close #20

### CODE REVIEW

File: dask_cugraph/tests/test_karate_csv.py [hunk 1]
```diff
@@ -38,15 +40,16 @@ def test_pagerank():
...
     print("CALLING DASK MG PAGERANK")
     pr = dcg.mg_pagerank(ddf)
-    print(pr)
     res_df = pr.compute()
-    #res_df = f_pr.sort_values('pagerank')
-    #print(len(res_df))
-    for c in range(len(res_df)):
-        print(res_df['vertex'][c], res_df['pagerank'][c])
+    err = 0
+    tol = 1.0e-02
```
Comment (afender) [inline dask_cugraph/tests/test_karate_csv.py:48]: I think decreasing `tol` to `1.0e-05` and adding a number of iteration of _at least_ `50` to  `dcg.mg_pagerank` would make the test stronger.

File: dask_cugraph/tests/test_karate_csv.py [hunk 2]
```diff
@@ -38,15 +40,16 @@ def test_pagerank():
...
+    for i in range(len(res_df)):
+        if(abs(res_df['pagerank'][i]-nx_pr[i][1]) > tol*1.1):
+            err = err + 1
+    print("Mismatches:", err)
+    assert err < (0.01*len(res_df)) 
 
-    assert 1==0
+    #assert 1==0
```
Comment (afender) [inline dask_cugraph/tests/test_karate_csv.py:55]: should be deleted

File: dask_cugraph/tests/test_hibench_small.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    import dask_cudf
+    import dask_cugraph.pagerank as dcg
+    input_df = cudf.DataFrame()
+    print("Read Input Data.")
+    ddf = dcg.read_csv(input_data_path, delimiter='\t', names=['src', 'dst'], dtype=['int32', 'int32'])
+    print("CALLING DASK MG PAGERANK")
+    pr = dcg.mg_pagerank(ddf)
+    res_df = pr.compute()
+    err = 0
+    tol = 1.0e-02
```
Comment (afender) [inline dask_cugraph/tests/test_hibench_small.py:49]: I think decreasing `tol` to `1.0e-05` and adding a number of iteration of _at least_ `50` to  `dcg.mg_pagerank` would make the test stronger.

**Review by afender — APPROVED** [review]
Looks good. I'd just recommend updating the test parameters

**Comment by afender** [comment]
Please add "close #<issue_number>" to the description.

**Comment by Iroy30** [comment]
close #20

**Comment by afender** [comment]
Should imports be at the top of the files or at least outside the function?

**Comment by afender** [comment]
close statement should be in the issue description, it doesn't close otherwise

File: dask_cugraph/tests/test_karate_csv.py (hunk 1 above)
Comment (Iroy30) [reply dask_cugraph/tests/test_karate_csv.py:48]: the max iter is 10 currently in cugraph wrapper, max iter is not included in the api for dask_cugraph call.

Comment (Iroy30) [reply dask_cugraph/tests/test_karate_csv.py:48]: the api needs to include max iter and alpha too

Comment (afender) [reply dask_cugraph/tests/test_karate_csv.py:48]: let's expose this in the API then

Comment (afender) [reply dask_cugraph/tests/test_karate_csv.py:48]: will be addressed in #30

**Pull request merged**